public/resources/core/js/services/billing-account.service.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * @author Francisco de la Vega <fdelavega@conwet.com>
+ *         Jaime Pajuelo <jpajuelo@conwet.com>
+ *         Aitor Magán <amagan@conwet.com>
+ */
+
+
+(function () {
+
+    'use strict';
+
+    angular
+        .module('app')
+        .factory('BillingAccount', BillingAccountService);
+
+    function BillingAccountService($q, $resource, URLS, User, CustomerAccount, Customer) {
+        var BillingAccount = $resource(URLS.BILLING_MANAGEMENT + '/billingAccount/:id', {}, {
+            update: {method: 'PUT'},
+            updatePartial: {method: 'PATCH'}
+        });
+        BillingAccount.prototype.getEmailAddress = function getEmailAddress() {
+            return this.customerAccount.customer.getEmailAddress();
+        };
+        BillingAccount.prototype.getPostalAddress = function getPostalAddress() {
+            return this.customerAccount.customer.getPostalAddress();
+        };
+        BillingAccount.prototype.getTelephoneNumber = function getTelephoneNumber() {
+            return this.customerAccount.customer.getTelephoneNumber();
+        };
+        BillingAccount.prototype.serialize = function serialize() {
+            return {
+                id: this.id,
+                href: this.href
+            };
+        };
+
+        var EVENTS = {
+        };
+
+        var TYPES = {
+        };
+
+        var TEMPLATES = {
+            BILLING_ACCOUNT: {
+                name: '',
+                state: 'Defined',
+                customerAccount: {},
+                relatedParty: []
+            }
+        };
+
+        return {
+            EVENTS: EVENTS,
+            TEMPLATES: TEMPLATES,
+            TYPES: TYPES,
+            search: search,
+            create: create,
+            detail: detail,
+            launch: launch
+        };
+
+        function search(filters) {
+            var deferred = $q.defer();
+            var params = {};
+
+            BillingAccount.query(params, function (billingAccounts) {
+                detailCustomerAccount(billingAccounts).then(function () {
+                    deferred.resolve(billingAccounts);
+                }, function (argument) {
+                    deferred.reject(response);
+                });
+            }, function (response) {
+                deferred.reject(response);
+            });
+
+            return deferred.promise;
+        }
+
+        function create(data) {
+            var deferred = $q.defer();
+
+            Customer.create(data.customerAccount.customer).then(function (customer) {
+                data.customerAccount.customer = customer;
```

Comment:
Why don't you create a new object just with the information that you want to introduce in the Customer Account resource?